README.md

```diff
@@ -50,25 +50,73 @@ You should start 4 instances parallel.
 Instance 1
 
 ```
+First run
+brew install librdkafka
+
+then
+make install
+
+To view brew services to make sure mosquitto has started
+brew services list
+
+then run
+git branch
+git log
+git pull
+
+then
 mosquitto -c config/mosquitto.conf
+
+and
+brew services start mosquitto
```

Comment:
this command is not necessary as `mosquitto -c config/mosquitto.conf` will take care of this.